### DISCUSSION TITLE
download webglfudamentals

### DISCUSSION THREAD

**Discussion by letochagone** [gfxfundamentals/webgl-fundamentals#425] [Q&A] [answered]
I think I've asked this question before, but I don't remember :
I would like to download the WebGLfundamentals site to: 1/ have it offline and 2/ add my small examples in the WebGL State Diagram part

**Comment by greggman** [comment]
you can either build it

https://github.com/gfxfundamentals/webgl-fundamentals#to-build

or you can download the current version of the site here

https://github.com/gfxfundamentals/webgl-fundamentals/archive/refs/heads/gh-pages.zip

**Reply by letochagone** [reply]
ok. I forgot to ask the same for webgl2 (the .zip current version of the site)

**Comment by greggman** [comment, marked as answer]
https://github.com/gfxfundamentals/webgl2-fundamentals/archive/refs/heads/gh-pages.zip
